highfidelity/hifi-content
# Hub content

## Test Plan
1. Load in "DomainContent/Hub/models/pill.fbx"
2. Set this as the server script: "DomainContent/Hub/pill/spawnPills.js"
Set this as the client script: "DomainContent/Hub/pill/swallowFXPill.js"
3. In desktop mode, click on the pill
4. You should hear a swallow effect, a "space ambient" effect, and your whole visual field should change to a randomly selected effect (available effects: WIREFRAME, Depth, Albedo, Normal, Metallic, Emissive) and the pill should disappear
5. While the effect is activated, nothing should happen if you click the pill again
6. It should reset to normal in 10 seconds
7. In HMD mode, grab inside the pill and pull out the clone pill and bring it up to your face to swallow
8. You should hear a swallow effect, a "space ambient" effect, and your visual field should change and the pill should disappear like in desktop mode
9. If you grab another pill and eat it, you should see a new effect and hear a new swallow effect and "space ambient" effect
10. Your visual field should return to normal after 10 seconds after the initial pill was eaten

### To be added in future iterations

- an original script spawner
- made a wearable
